[2022-02-28 14:59:18,395] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-14T00:00:00+00:00 [queued]>
[2022-02-28 14:59:18,404] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-14T00:00:00+00:00 [queued]>
[2022-02-28 14:59:18,404] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 14:59:18,404] {taskinstance.py:1244} INFO - Starting attempt 1 of 3
[2022-02-28 14:59:18,404] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 14:59:18,414] {taskinstance.py:1264} INFO - Executing <Task(BranchPythonOperator): new_comic_available> on 2022-01-14 00:00:00+00:00
[2022-02-28 14:59:18,416] {standard_task_runner.py:52} INFO - Started process 34442 to run task
[2022-02-28 14:59:18,423] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'comic_app_v2', 'new_comic_available', 'scheduled__2022-01-14T00:00:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/comic_app_v2.py', '--cfg-path', '/var/folders/jb/grqrlwkd311flwk_zt30r7wr0000gn/T/tmpfvj9e_la', '--error-file', '/var/folders/jb/grqrlwkd311flwk_zt30r7wr0000gn/T/tmp27c0yssc']
[2022-02-28 14:59:18,425] {standard_task_runner.py:77} INFO - Job 140: Subtask new_comic_available
[2022-02-28 14:59:18,473] {logging_mixin.py:109} INFO - Running <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-14T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-02-28 14:59:18,514] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Adam
AIRFLOW_CTX_DAG_ID=comic_app_v2
AIRFLOW_CTX_TASK_ID=new_comic_available
AIRFLOW_CTX_EXECUTION_DATE=2022-01-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-14T00:00:00+00:00
[2022-02-28 14:59:18,519] {logging_mixin.py:109} INFO - 跟紀錄比較，有沒有新連載？
[2022-02-28 14:59:18,519] {python.py:175} INFO - Done. Returned value was: yes_generate_notification
[2022-02-28 14:59:18,520] {skipmixin.py:140} INFO - Following branch yes_generate_notification
[2022-02-28 14:59:18,520] {skipmixin.py:171} INFO - Skipping tasks ['no_do_nothing']
[2022-02-28 14:59:18,544] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=comic_app_v2, task_id=new_comic_available, execution_date=20220114T000000, start_date=20220228T065918, end_date=20220228T065918
[2022-02-28 14:59:18,567] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-28 14:59:18,603] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-28 15:05:49,263] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-14T00:00:00+00:00 [queued]>
[2022-02-28 15:05:49,277] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-14T00:00:00+00:00 [queued]>
[2022-02-28 15:05:49,277] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 15:05:49,277] {taskinstance.py:1244} INFO - Starting attempt 1 of 3
[2022-02-28 15:05:49,277] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 15:05:49,289] {taskinstance.py:1264} INFO - Executing <Task(BranchPythonOperator): new_comic_available> on 2022-01-14 00:00:00+00:00
[2022-02-28 15:05:49,292] {standard_task_runner.py:52} INFO - Started process 35021 to run task
[2022-02-28 15:05:49,299] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'comic_app_v2', 'new_comic_available', 'scheduled__2022-01-14T00:00:00+00:00', '--job-id', '216', '--raw', '--subdir', 'DAGS_FOLDER/comic_app_v2.py', '--cfg-path', '/var/folders/jb/grqrlwkd311flwk_zt30r7wr0000gn/T/tmp1pb9dfs8', '--error-file', '/var/folders/jb/grqrlwkd311flwk_zt30r7wr0000gn/T/tmpkh6pa5ha']
[2022-02-28 15:05:49,302] {standard_task_runner.py:77} INFO - Job 216: Subtask new_comic_available
[2022-02-28 15:05:49,350] {logging_mixin.py:109} INFO - Running <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-14T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-02-28 15:05:49,389] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Adam
AIRFLOW_CTX_DAG_ID=comic_app_v2
AIRFLOW_CTX_TASK_ID=new_comic_available
AIRFLOW_CTX_EXECUTION_DATE=2022-01-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-14T00:00:00+00:00
[2022-02-28 15:05:49,393] {logging_mixin.py:109} INFO - 跟紀錄比較，有沒有新連載？
[2022-02-28 15:05:49,394] {python.py:175} INFO - Done. Returned value was: yes_generate_notification
[2022-02-28 15:05:49,394] {skipmixin.py:140} INFO - Following branch yes_generate_notification
[2022-02-28 15:05:49,394] {skipmixin.py:171} INFO - Skipping tasks ['no_do_nothing']
[2022-02-28 15:05:49,415] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=comic_app_v2, task_id=new_comic_available, execution_date=20220114T000000, start_date=20220228T070549, end_date=20220228T070549
[2022-02-28 15:05:49,440] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-28 15:05:49,473] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
